pkg/cmd/trait_bind.go

```diff
@@ -62,12 +65,14 @@ func AddTraitPlugins(parentCmd *cobra.Command, c client.Client, ioStreams cmduti
 		pluginCmd.SetOut(o.Out)
 		for _, v := range tmp.Parameters {
 			pluginCmd.Flags().StringP(v.Name, v.Short, v.Default, v.Usage)
-			if v.Required {
-				pluginCmd.MarkFlagRequired(v.Name)
-			}
+			// These flags are exclusive to `--detach`, so comments the lines.
```

Comment:
according to this case, maybe `<workloadtype>:detach` will be better.

Reply:
> according to this case, maybe `rudr <workloadtype>:detach` will be better.

`rudr <workloadtype>:detach <traittype> -a <applicationName>`?

Reply:
I'm sorry, `rudr <traittype>:detach  applicationName`

Reply:
Done